tests/test_config_section_ahv.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
```

Comment:
Do we need this against main?

Reply:
It is consistent with the other modules. Perhaps we make it uniform and then remove them all at the same time.